cfme/tests/infrastructure/test_advanced_search_host.py

```diff
@@ -198,7 +197,7 @@ def test_quick_search_without_host_filter(host_collection, request, hosts, hosts
     request.addfinalizer(view.entities.search.clear_simple_search)
     view.flash.assert_no_error()
     # Check it is there
-    all_hosts_visible = host_collection.all(infra_provider)
+    all_hosts_visible = view.entities.get_all()
```

Comment:
:+1: nice
Optional:
you can use `view.entities.entity_names` it will return you directly `list` of names of `all_visible_hosts`. Now assert will easy for you just need to check `median_host in all_visible_hosts`